nixos/modules/virtualisation/docker.nix

```diff
@@ -150,6 +150,10 @@ in
 
   config = mkIf cfg.enable (mkMerge [{
       boot.kernelModules = [ "bridge" "veth" ];
+      boot.kernel.sysctl = {
+        "net.ipv4.conf.all.forwarding" = mkOverride 99 true;
+        "net.ipv4.conf.default.forwarding" = mkOverride 99 true;
```

Comment:
https://docs.docker.com/network/bridge/#enable-forwarding-from-docker-containers-to-the-outside-world was the only upstream source I could find. I also checked by manually restarting docker after doing a system rebuild and comparing sysctl before and after and these two options are definitely set by docker on the host.

https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt specifies what they mean if anybody wants to read that.